### ISSUE TITLE
Troubleshooting DOM Changes - more detail needed please

### ISSUE THREAD

**Issue by Chris-Karona** [mendix/docs#4035]
In the section on 'Streamlined Custom Themes', it says download the files for legacy UI themes.  That's very helpful but it doesn't list which files are needed or where to put them.  I shouldn't have to find out by trial and error. 
I have a legacy app which started life as Mx V3.  There is no business case for making any more changes than absolutely necessary but we have to migrate from Mx6 to Mx7 because Mx6 is now unsupported. Since Mx7 does not have much life left in it we want to migrate to Mx8 at the same time.  I expect other customers will face this in the near future. 
https://docs.mendix.com/refguide8/migration-dom-issues

**Comment by ConnorLand** [comment]
Hi @Chris-Karona ,

My sincere apologies for your feedback item staying unaddressed for so long. After chatting with developers, I have found a bit more info to address questions like the ones you posed.

I've found some information on which exact files must be added to various styling folders when migrating into Mx8. Thus, I've added new details in the doc here: https://github.com/mendix/docs/commit/a5182e8d9b5569d64285ecfa7af9ea0bae3cf3c8

Your other feedback issues are being looked into, and we sincerely thank you for your engagement with the Mendix documentation and our products.

Best,

Connor
